ballerina/Module.md

```diff
@@ -93,4 +93,6 @@ bal run
 
 ## Examples
 
-The `Spotify` connector provides practical examples illustrating usage in various scenarios. 
\ No newline at end of file
+The `Spotify` connector provides practical examples illustrating usage in various scenarios. Explore these [examples](https://github.com/module-ballerinax-spotify/tree/main/examples/), covering the following use cases:
+
+1. [Get albums of artist in Spotify](https://github.com/ballerina-platform/module-ballerinax-spotify/tree/main/examples/get-artist-albums) - Get albums of artist in Spotify
```

Comment:
Here we are not showing only the album details, but also other details like genres, followers also. So shall we update the example name in here to reflect that?